cmake/everest-generate.cmake

```diff
@@ -494,7 +507,102 @@ function (ev_add_py_module MODULE_NAME)
             add_subdirectory(${MODULE_PATH})
         endif()
     else()
-        message(WARNING "C++ module ${MODULE_NAME} does not exist at ${MODULE_PATH}")
+        message(WARNING "Python module ${MODULE_NAME} does not exist at ${MODULE_PATH}")
+        return()
+    endif()
+
+    # FIXME (aw): this will override EVEREST_MODULES, might not what we want
+    set_property(
+        GLOBAL
+        PROPERTY EVEREST_MODULES ${EVEREST_MODULES}
+    )
+endfunction()
+
+function (create_rust_cargo_toml RUST_MODULE_LIST)
+    if(NOT ${EVEREST_ENABLE_RS_SUPPORT})
+        return()
+    endif()
+
+    set(OUTPUT_FILE_PATH "${CMAKE_BINARY_DIR}/modules/Cargo.toml")
```

Comment:
I would probably choose not the the `modules` folder, rather the `generated/rust-whatever..`

Reply:
The Cargo.toml of the workspace must be in the top level of all Cargo modules. Having it in modules also allows to defer the exact location of where the binary ends up and makes sure that all rust modules use a consistent set of dependencies. I advise leaving it at its current place.